Swat/SwatTableView.php

```diff
@@ -474,8 +494,13 @@ public function hasMessage()
 	public function getXhtmlColspan()
 	{
 		$count = 0;
-		foreach ($this->getVisibleColumns() as $column)
+		foreach ($this->getVisibleColumns() as $column) {
 			$count += $column->getXhtmlColspan();
+		}
+
+		if ($count < 2 && count($this->getSpanningColumns()) > 0) {
```

Comment:
@gauthierm what I have is a TableView with only spanning columns a group-by field. The way it currently is, the group-by header only spans one column and is squished. So we either need to:
 - check if any spanning-column has a title and set to `2`
 - not use a table-view that way, and instead use a replicator